base_search_mail_content/readme/INSTALL.rst

```diff
@@ -0,0 +1,6 @@
+This module depends on the module 'base_search_fuzzy' to ensure that
+searches on emails are based on indexes. Please read carefully the
+`install instructions <https://github.com/OCA/server-tools/blob/10.0/base_search_fuzzy/README.rst>`_.
```

Comment:
Not sure if we should link the README now that it is split... Maybe better the odoo apps site? Besides, it's v10 branch.